testsuites/CBLTester/upgrade_tests/test_cbl_upgrade.py

```diff
@@ -0,0 +1,133 @@
+import pytest
+import os
+
+from CBLClient.Database import Database
+from keywords.utils import log_info
+from CBLClient.Replication import Replication
+from keywords.MobileRestClient import MobileRestClient
+from CBLClient.Authenticator import Authenticator
+from libraries.testkit.cluster import Cluster
+from couchbase.n1ql import N1QLQuery
+from couchbase.bucket import Bucket
+from keywords.constants import SDK_TIMEOUT
+import time
+from CBLClient.MemoryPointer import MemoryPointer
+from keywords.couchbaseserver import CouchbaseServer
+
+
+@pytest.mark.listener
+@pytest.mark.upgrade_test
+def test_upgrade_cbl(params_from_base_suite_setup):
+    base_liteserv_version = params_from_base_suite_setup["base_liteserv_version"]
```

Comment:
can you add summary for test upgrade for all the steps that is doing in the test?

Reply:
added